### ISSUE TITLE
Replace Swig with Handlebars (or Nunjucks/similar)

### ISSUE THREAD

**Issue by simison** [Trustroots/trustroots#410]
There's an excellent PR merged in MEAN.js, so we can use that directly: https://github.com/meanjs/mean/pull/1461

Swig is outdated, little bit unsecure and no more maintained.

On our version Swig is passed trough this file: https://github.com/Trustroots/trustroots/blob/master/config/lib/render.js

**Comment by rpaggi** [comment]
I'll work on this issue!

**Comment by simison** [comment]
@rpaggi uuh, sorry, I dropped on this due busy times. :-/ I had a look at the code and the problem is that Handlebars doesn't offer direct access to `render()` the way Swig did (this might be worth double checking, I might be wrong). I'm not sure if it's worth time figuring out how to do direct rendering with Handlebars instead of just using something else?

While it would be really nice to keep in par with upstream MEAN.js, it might make sense to just use [Nunjucks](https://mozilla.github.io/nunjucks/) instead, as their [api](https://mozilla.github.io/nunjucks/api.html) seems to have really nice `render()` and `renderString()` methods. What do you think?

Bottom line is we just need something that's supported, since Swig has been abandoned long ago and has some security issues.

Sorry again for abandoning you on this! :cheese: :beers:
